### PR TITLE
Add roads project and example notebook

### PR DESCRIPTION
**Pull request by samredai**
### Summary

This adds the roads example data model as a YAML project as well as a notebook that goes through the process of creating a new user, loading the roads project, validating it, then deploying it. Since we're using the postgres container now, the demo DJ environment starts up as an fresh/empty deployment. Running this notebook deploys the project defined in the YAML files to quickly get the demo environment up and running.

### Test Plan

Ran `docker compose --profile=demo up` and ran the `DJ Projects Tutorial.ipynb` notebook.

- [ ] PR has an associated issue: #
- [ ] `make check` passes
- [ ] `make test` shows 100% unit test coverage

### CODE REVIEW

**Review by shangyian — APPROVED** [review]
Nice! Thanks for adding

File: notebooks/example_project/dj.yaml
Comment (agorajek) [inline notebooks/example_project/dj.yaml:1]: Love the simplicity here.

File: .gitignore [hunk 1]
```diff
@@ -120,3 +120,6 @@ client_secret*
 
 # random notebooks
 Untitled*
+
+#postgres
```
Comment (agorajek) [inline .gitignore:124]: nit: space between `#` and `postgres` .

**Review by agorajek — APPROVED** [review]
Oh cool. This is awesome.

File: .gitignore (hunk 1 above)
Comment (samredai) [inline .gitignore:124]:
```suggestion
# postgres
```

**Comment by CircArgs** [comment]
"Now I am become YAML, shatterer of worlds."

**Pull request merged**